lib/bucky/core/test_core/test_class_generator.rb

```diff
@@ -16,7 +16,13 @@ module TestClassGeneratorHelper
         def add_test_procedure(procedures)
           procedures.each.with_index(1) do |procedure, index|
             procedure[:proc] ||= ''.dup
-            puts procedure[:proc].insert(0, "  #{index}:") if procedure.key?(:proc)
+            # Check proc name already have a index (0~999)
+            # ex:) 100: proc name
+            if procedure[:proc].match(/^[0-9]{,3}:/)
```

Comment:
Please tell me about this condition.
I tested, but returned `nil`.
This condition is correct?

```
proc = '  1:click'
=> "  1:click"
proc.match(/^[0-9]{,3}:/)
=> nil
```

Reply:
There will be no space in front of the index.
Because they will be delete in here.
https://github.com/lifull-dev/bucky-core/blob/master/lib/bucky/test_equipment/user_operation/user_operator.rb#L37

It should be like this: 
```
irb(main):001:0> proc='1:click'
=> "1:click"
irb(main):002:0> proc.match(/^[0-9]{,3}:/)
=> #<MatchData "1:">
```

Reply:
But it may get some bugs when not raising exception.
I'll fix it!